.travis.yml

```diff
@@ -41,16 +41,21 @@ cache:
 addons:
   apt:
     sources:
-    - boost-latest
     - ubuntu-toolchain-r-test
     packages:
-    - libboost1.55-all-dev
-    - libboost1.55-tools-dev
     - python2.7-dev
     - g++-5
 
 before_install:
-
+  # on linux, pull down a recent version of boost and build boost-build
+  - 'if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+    wget https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.zip;
```

Comment:
are you willing to increase the minimal boost version? is there something you need in particular?